internal/bot.go

```diff
@@ -98,22 +101,25 @@ func (bot Bot) runBot() {
 			delete(chatIDs, chatID)
 
 		case <-ticker.C:
-			for chatID := range chatIDs {
-				bot.sendReminder(chatID)
+			// skip weekends
+			if !strings.Contains(weekends, bot.time.Weekday().String()) {
```

Comment:
do a proper skipping on num value of friday and saturday (check the consts of weekedays and their numerical value)